### PR TITLE
feat: add maximum reports to store

### PR DESCRIPTION
**Pull request by gaetansenn**
Add the possibility to set the maximum number of reports to save disk space.

### CODE REVIEW

**Comment by gaetansenn** [comment]
Look's like your eslint from @netsells/eslint-config is not setup well for this module.

File: lib/ReportHistory.js [hunk 1]
```diff
@@ -52,14 +57,21 @@ class ReportHistory {
...
     store(report) {
-        lastReportOffset++;
-
         if (!fs.existsSync(this.storageDirectory)) {
             fs.mkdirSync(this.storageDirectory);
         }
 
         fs.writeFileSync(this.newReportFile, JSON.stringify(report, null, 4));
 
+        const files = fs.readdirSync(this.storageDirectory)
```
Comment (samturrell) [inline lib/ReportHistory.js]: Looks like we're using this twice. Might be worth moving to another method/getter called something like `storedReports`

**Review by samturrell — CHANGES_REQUESTED** [review]
Looks good @gaetansenn!

File: lib/ReportHistory.js (hunk 1 above)
Comment (gaetansenn) [reply lib/ReportHistory.js]: Done ;)

**Review by samturrell — APPROVED** [review]

**Comment by samturrell** [comment]
`3.1.0` tagged and published, thanks @gaetansenn 🎉

**Pull request merged**